test/embind/test_return_value_policy.cpp

```diff
@@ -78,26 +78,44 @@ void assertAllCountsZero() {
 struct ValueHolder {
   ValueHolder() {};
   Value value;
+  Value* valuePtr;
+
   Value* getPtr() {
     return &value;
   }
-  Value* getNewPtr() {
-    return new Value();
-  }
   Value getValue() {
     return value;
   }
   Value& getRef() {
     return value;
   }
+
+  void set(Value v) {
+    value = v;
+  }
+  Value get() const {
+    return value;
+  }
+  void initializePtr() {
+    valuePtr = new Value();
+  }
+  Value* getNewPtr() const {
```

Comment:
I see this function moved and became `() const {` - was that just a cleanup or is it necessary?

Reply:
Embind requires getter function's to be const. We could have a const and non-const version for testing functions/properties, but doesn't really seem worth it.

Reply:
Makes sense. Is that a new change with this PR, though? That's what I'm trying to understand, as the function seems to have worked ok as non-const before.

Reply:
`const` is not a new requirement for getters. I added the `const` here since I now use `value_holder` to also test a getter property.  `value_holder` calls `ValueHolder.getNewPtr` so it must also be const.

Reply:
I see, thanks.